cockatrice/src/dialogs/dlg_settings.cpp

```diff
@@ -582,6 +587,7 @@ void UserInterfaceSettingsPage::retranslateUi()
 {
     generalGroupBox->setTitle(tr("General interface settings"));
     doubleClickToPlayCheckBox.setText(tr("&Double-click cards to play them (instead of single-click)"));
+    clickPlaysAllSelectedCheckBox.setText(tr("&Clicking plays all selected cards (instead of just the clicked card)"));
```

Comment:
I don't like the wording, but I'm not sure what would be better. @ebbit1q any ideas?